test/web-platform-tests/to-upstream/css/css-box/negative-margin.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Negative margins</title>
+<link rel="help" href="https://drafts.csswg.org/css-box/#margin">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<!-- Regression test for https://github.com/jsdom/cssstyle/issues/222 -->
+
+<div id="div"></div>
+
+<script>
+"use strict";
+const div = document.getElementById("div");
+
+test(() => {
+  div.style.marginTop = "-10px";
+  assert_equals(div.style.getPropertyValue("margin-top"), "-10px");
+  div.style.marginTop = null;
```

Comment:
```suggestion
test(t => {
  t.add_cleanup(() => { div.style.marginTop = null; });
  div.style.marginTop = "-10px";
  assert_equals(div.style.getPropertyValue("margin-top"), "-10px");
```

here and below. This is best practice to ensure that one test failing does not cause all other tests to fail.

Reply:
Done